.github/workflows/builds.yml

```diff
@@ -22,6 +22,9 @@ jobs:
       uses: cake-build/cake-action@v1.4.0
       with:
         target: Test
+
+    - name: EditorConfig-Action
+      uses: zbeekman/EditorConfig-Action@v1.1.1
```

Comment:
It would be great to check only lint files changed in the pull request. Please add a proper env. variable

Reply:
I'm having trouble running the new workflow locally with nektos/act.
I want to test it before updating the PR, any recommendations?